external/tclap/CmdLine.h

```diff
@@ -504,7 +504,8 @@ inline void CmdLine::parse(std::vector<std::string>& args)
 	}
 
 	if (shouldExit)
-		exit(estat);
+        throw(CmdLineParseException("Command line argument exception", gismo::util::to_string(estat))); // G+Smo
```

Comment:
It is supposed to be a clean exit; we might consider (re-)throwing ExitException.